README.md

```diff
@@ -1,2 +1,71 @@
-# Ed-Fi-Kafka-Connect
-Custom Kafka Connect transform for the Ed-Fi Data Management Service
+# Ed-Fi Transformations for Apache Kafka® Connect
+
+[Single Message Transformations
+(SMTs)](https://kafka.apache.org/documentation/#connect_transforms) for Apache
+Kafka Connect.
+
+## Transformations
+
+See [the Kafka
+documentation](https://kafka.apache.org/documentation/#connect_transforms) for
+more details about configuring transformations on how to install transforms.
+
+### `GenerateIndexFromResource`
+
+This transformation builds an index based on a group of values contained in the
+body of the result, separated by $.
+
+- `org.ed-fi.kafka.connect.transforms.GenerateIndexFromResource`
+  - works on values.
```

Comment:
During sprint planning, we realized that this should be DMS specific. Maybe `GenerateIndexFromDmsResource`?